release-index.yaml

```diff
@@ -49,69 +49,69 @@ releases:
         version: 0d2b3965c813cd3a39ceedacd97fa2eee8760074
   - rc_name: rc--2024-07-03_23-01
     versions:
-      - # Successful qualification pipeline: https://gitlab.com/dfinity-lab/core/release/-/pipelines/1360352158
-        name: base
+      # Successful qualification pipeline: https://gitlab.com/dfinity-lab/core/release/-/pipelines/1360352158
+      - name: base
```

Comment:
Had to make these adjustments because the tool automatically formatted them. Its basically just moving comments around.